sympy/core/power.py

```diff
@@ -514,7 +518,7 @@ def _eval_is_positive(self):
 
     def _eval_is_extended_positive(self):
         from sympy import log
-        if self.base == self.exp:
+        if self.base == self.exp and not self.base.is_imaginary:
```

Comment:
This was causing `(I**I).is_extended_positive` to return `None` instead of `True`

Reply:
You need to be careful using `not` in fuzzy logic. Remember that `is_imaginary` can be `True`, `False`, or `None`.

Do we just need to change the `elif` below to `if`?

Reply:
Alright I did the necessary change.